Core/include/Acts/Seeding/SeedFilter.ipp

```diff
@@ -52,16 +74,21 @@ void SeedFilter<external_spacepoint_t>::filterSeeds_2SpFixed(
     float invHelixDiameter = invHelixDiameterVec[i];
```

Comment:
the comment 3 lines higher should be updated to reflect that this is NOT yet fulfil the "quality seed" criteria but is a weaker requirement

Reply:
done